applications/NXtomo.nxdl.xml

```diff
@@ -27,14 +27,20 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
-
 	<symbols>
-		<doc>These symbols will be used below to coordinate datasets with the same shape.</doc>
-		<symbol name="nFrames"><doc>number of frames</doc></symbol>
-		<symbol name="xsize"><doc>number of pixels in X direction</doc></symbol>
-		<symbol name="ysize"><doc>number of pixels in Y direction</doc></symbol>
+		<doc>
```

Comment:
indentation _looks_ off on github but that's just the XML tags, probably does not affect the code that pulls the rst text for documentation.